webpack/assets/javascripts/react_app/components/HostDetails/DetailsCard/index.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import {
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  Card,
+  CardHeader,
+  CardTitle,
+  CardBody,
+  CardActions,
+  ClipboardCopy,
+} from '@patternfly/react-core';
+import { UserIcon } from '@patternfly/react-icons';
+import { foremanUrl } from '../../../common/helpers';
+import PowerStatusDropdown from './PowerStatusDropdown';
+import { translate as __ } from '../../../common/I18n';
+
+const DetailsCard = ({
+  id: hostID,
+  ip,
+  mac,
+  uuid,
+  comment,
+  owner_id: ownerID,
+  owner_name: ownerName,
+  hostgroup_name: hostgroupName,
+  hostgroup_id: hostgroupID,
+}) => (
+  <Card isHoverable>
+    <CardHeader>
+      <CardTitle>{__('Details')}</CardTitle>
+      <CardActions>
+        <PowerStatusDropdown hostID={hostID} />
+      </CardActions>
+    </CardHeader>
+    <CardBody>
+      <DescriptionList
+        isInlineGrid
+        columnModifier={{
+          default: '2Col',
+        }}
+      >
+          
+        <DescriptionListGroup>
+          <DescriptionListTerm>{__('IP Address')}</DescriptionListTerm>
```

Comment:
Please add IPv6 too.

Reply:
Thanks, addded.
ipv6 is shown by default if exists, in this case, the ipv4 (if exists) will be shown in the details tab.